nat-lab/tests/test_netcat.py

```diff
@@ -0,0 +1,55 @@
+import asyncio
+import pytest
+from contextlib import AsyncExitStack
+from helpers import SetupParameters, setup_environment
+from utils.connection_util import ConnectionTag, new_connection_by_tag
+from utils.netcat import NetCatServer, NetCatClient
+
+TEST_STRING = "test_data"
+PORT = 12345
+
+
+@pytest.mark.asyncio
+@pytest.mark.timeout(30)
+@pytest.mark.parametrize(
+    "setup_params",
+    [
+        pytest.param(
+            SetupParameters(
+                connection_tag=ConnectionTag.MAC_VM,
+            ),
+            marks=pytest.mark.mac,
+        ),
+    ],
+)
```

Comment:
You can just get rid of `setup_params` parameter, and hardcode MAC_VM the same way you did for CONE_CLINET

Reply:
Then it's just gonna be an easy copy paste to add windows :D